ManatiCR/manatime
# 54: Remove manatime theme activator.

**Steps to test**
- Check if manatime theme was enabled.
- Check if site_template was enabled in '/admin/structure/pages'.

[issue link](https://github.com/ManatiCR/manatime/issues/54)
